src/utils/formik-helpers.ts

```diff
@@ -454,3 +455,28 @@ export const getTouchedTabFields = (
   const mergedFields = deepmerge.all(fieldsToTouchOnMount);
   return mergedFields;
 };
+
+export const checkIfTabErrors = (tabErrors: TabErrors) => {
+  let errors = 0;
+  Object.entries(tabErrors).forEach(([key, value]) => {
+    if (value.length > 0) {
+      ++errors;
+      return;
+    }
+  });
+  return errors > 0;
+};
+
+export const validateForm = (values: Registration): FormikErrors<Registration> => {
```

Comment:
Vi har mange forms, så kanskje tydeliggjøre hvilken form som valideres her?
```suggestion
export const validateRegistrationForm = (registration: Registration): FormikErrors<Registration> => {
```